media-gfx/qview/qview-3.0.recipe

```diff
@@ -1,20 +1,28 @@
 SUMMARY="Minimalist image viewer"
-DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+DESCRIPTION="qView is an image viewer designed with minimalism and usability \
+in mind. \
+qView was designed from the very start to be as visually minimal and space \
+efficient as possible. No cluttered interface, just your image and a titlebar.\
+qView opens practically instantly, and switching images is just as quick, all \
+while having low memory and CPU usage. \
+With features such as animated GIF controls, file history, rotation/mirroring,\
+ and multithreaded preloading, minimalism doesn't have to be a compromise.
```

Comment:
To be consistent, the space should go before the "`\`" of the line above.
Also the closing quote " should go at the end of this line.